.github/workflows/unit-tests.yml

```diff
@@ -28,12 +28,12 @@ jobs:
         include:
           - php: '7.2'
           - php: '7.4'
-          - php: '8.1'
+          - php: '8.2'
             mode: high-deps
-          - php: '8.1'
-            mode: low-deps
           - php: '8.2'
-            #mode: experimental
+            mode: low-deps
+          - php: '8.3'
+            mode: experimental
```

Comment:
Let's keep this commented out, it's green.

Reply:
Done. But we will need to enable it again on 6.2 and above because of Laminas Code.